modules/apps/object/object-api/src/main/java/com/liferay/object/system/util/SystemObjectDefinitionManagementChecker.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.system.util;
+
+import com.liferay.batch.engine.util.BatchEngineThreadLocal;
+import com.liferay.portal.kernel.util.SetUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.util.Collections;
+import java.util.Set;
+
+/**
+ * @author Gabriel Albuquerque
+ */
+public class SystemObjectDefinitionManagementChecker {
+
+	public static boolean isInvokerBundleAllowed() {
+		String bundleNamespace =
```

Comment:
What do you think about rename this variable "bundleNamespace" to "invokerBundleNamespace" ?